model/ftn/w3wavemd.ftn

```diff
@@ -496,7 +496,7 @@
 !
 ! 0.a Set pointers to data structure
 !
-!/OMPX      FLOMP  = .TRUE.
+!!/OMPX      FLOMP  = .TRUE.
```

Comment:
Is FLOMP redundant now (i.e. always False)? It is used in conditional statements on lines 579, 1733, 1786 and 2606.

Reply:
This was a typo left over from debugging. It has been changed back to the original function.